### PR TITLE
Enable voting for very old accounts with few valid games

### PR DESCRIPTION
**Pull request by Brutus5000**
(e.g. single player fanatics).
Lex Uveso

### CODE REVIEW

File: src/main/java/com/faforever/api/voting/VotingService.java [hunk 1]
```diff
@@ -100,7 +101,9 @@ public void saveVote(Vote vote, Player player) {
       errors.add(new Error(ErrorCode.VOTE_ALREADY_ENDED, subject.getEndOfVoteTime()));
     }
 
-    if (gamesPlayed < subject.getMinGamesToVote()) {
+    boolean accountQualifiedBySteamAndAge = player.getSteamId() != null && OffsetDateTime.now().minusYears(ACCOUNT_AGE_YEARS_VOTE_QUALIFIED).isAfter(player.getCreateTime());
+
+    if (!accountQualifiedBySteamAndAge && gamesPlayed < subject.getMinGamesToVote()) {
       errors.add(new Error(ErrorCode.NOT_ENOUGH_GAMES, gamesPlayed, subject.getMinGamesToVote()));
```
Comment (bukajsytlos) [inline src/main/java/com/faforever/api/voting/VotingService.java:107]: Shouldn't we change error message too?

**Review by bukajsytlos — COMMENTED** [review]
No tests? :(

File: src/main/java/com/faforever/api/voting/VotingService.java (hunk 1 above)
Comment (Brutus5000) [reply src/main/java/com/faforever/api/voting/VotingService.java:107]: No, **if** the error occurs its still correct. It just occurs in less cases.

Comment (bukajsytlos) [reply src/main/java/com/faforever/api/voting/VotingService.java:107]: wah, you are right

**Review by bukajsytlos — APPROVED** [review]

**Pull request merged**